spec/unit/navigation/navigation.spec.js

```diff
@@ -66,6 +67,39 @@ describe('navigation toggle', function () {
     assert.equal(isVisible(nav), false);
   });
 
+  it('focuses the close button when the menu button is clicked', function () {
+    menuButton.click();
+    assert.equal(document.activeElement, closeButton);
+  });
+
+  it('focuses the menu button when the close button is clicked', function () {
+    menuButton.click();
+    closeButton.click();
+    assert.equal(document.activeElement, menuButton);
+  });
+
+  it('collapses nav if needed on window resize', function () {
+    menuButton.click();
+    sinon.stub(closeButton, 'getBoundingClientRect').returns({ width: 0 });
+    try {
+      window.dispatchEvent(new CustomEvent('resize'));
+    } finally {
+      closeButton.getBoundingClientRect.restore();
```

Comment:
Ah nice, I didn't know about [sinon sandboxes](http://sinonjs.org/releases/v1.17.7/sandbox/)!  Thanks, I'll convert this to use them.